.travis/run_all_tests.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+STATUS_CODE=0
+
+####################
+echo "Check: go vet"
+go vet ./...
+RESULT=$?
+if [ $RESULT -ne 0 ]; then
+    echo "Check 'go vet': FAIL"
+    STATUS_CODE=1
+else
+	echo "Check 'go vet': OK"
```

Comment:
Should this indentation line up with the 'if' block? (also on lines 24, 37, and 49).